src/equations/numerical_fluxes.jl

```diff
@@ -216,6 +216,7 @@ left and right states `u_ll, u_rr`, usually based only on the local wave speeds
   [DOI: 10.1137/1025002](https://doi.org/10.1137/1025002)
 """
 function min_max_speed_naive end
+function min_max_speed_chen_noelle end
```

Comment:
Please add a docstring with a reference. If you have docstrings for all individual methods, you don't need to declare the function here.

Reply:
This function has a docstring in the `shallow_water_2d.jl` file. Is there a way to reference that docstring via [`min_max_speed_chen_noelle´](@ref) or would this not work because these functions have the same name so the documenter will not which function to jump to?

Reply:
I don't understand this question. If you have a function `min_max_speed_chen_noelle` with one method implementation that has already a docstring, then there is no need to add any docstring here - or to declare the function here.

Reply:
I believe it is an inheritance issue. We need to declare an empty function here for `min_max_speed_chen_noelle` such that the dimension agnostic, Chen and Noelle variant of `FluxHLL` knows what about this wave speed estimate routine.

Reply:
There is already a generic, but special, variant of `FluxHLL` for shallow water such that we do not add dissipation to the bottom topography variable. So, this circuitous way of defining the `flux_hll_chen_and_noelle` within the `numerical_flux.jl` was to avoid code duplication (I think). @svengoldberg would be able to comment on this.

If it makes things less confusing, we could just write a new `function flux_hll_chen_and_noelle`that would live in the respective shallow water files. However, this might lead to a bit of code duplication.

Reply:
Aha, now I see. In this case, please move the declaration `function min_max_speed_chen_noelle end` closer to `const flux_hll_chen_noelle = FluxHLL(min_max_speed_chen_noelle)` and describe the reason why you need to declare it first in a comment. Since it appeared directly after `function min_max_speed_naive end`, which we used to add a docstring, I assumed you wanted to do the same but just forgot it

Reply:
Okay, that is easy enough. Thanks!

Reply:
Thanks for being patient with a dump reviewer :slightly_smiling_face:

Reply:
Done. I moved this empty function definition and added a small comment block to discuss why it is there.